core/src/main/java/io/kestra/core/models/HasUID.java

```diff
@@ -0,0 +1,21 @@
+package io.kestra.core.models;
+
+/**
+ * A UID is used to uniquely identify an entity across an entire Kestra's cluster.
+ * <p>
+ * A UID is either a unique random ID or composed of a combination of entity properties such as
+ * the associated tenant, namespace, and identifier.
+ * <p>
+ * For Kestra's queuing mechanism the UID can be used as routing/or partitioning key.
+ */
+public interface HasUID {
```

Comment:
I prefer this name but we usually name these kind of things UIDInterface.
But I don't like naming interface interface as, well, it's redundant. So I'm OK with it but I had to notice that ;).